scilpy/image/resample_volume.py

```diff
@@ -46,23 +46,32 @@ def resample_volume(img, ref=None, res=None, iso_min=False, interp='lin',
         Resampled image.
     """
     data = img.get_fdata(dtype=np.float32)
```

Comment:
@arnaudbore Data is always loaded as float32, which breaks masks (can't be loaded with `load_data_as_mask` anymore) if you try to downsample one. What's the solution to this ?

Reply:
Do you think the resampling algorithm could be different with masks apart from the loading part ?

Reply:
Add zoom in Parameters

Reply:
You could add a boolean to say if it's a mask or not.

Reply:
Or find the data type and load accordingly

Reply:
Found how to load data as-is here: https://nipy.org/nibabel/reference/nibabel.dataobj_images.html?highlight=get_fdata#nibabel.dataobj_images.DataobjImage.get_data

Will update.

Reply:
Donesies in 7904f52